kadi/commands/commands.py

```diff
@@ -135,8 +135,8 @@ def get_cmds_from_backstop(backstop, remove_starcat=False):
 
     n_bs = len(bs)
     out = {}
-    # Set idx to max (2**16 -1) so it does not match any real idx
-    out['idx'] = np.full(n_bs, fill_value=65535, dtype=np.uint16)
+    # Set idx to max (2**32 -1) so it does not match any real idx
+    out['idx'] = np.full(n_bs, fill_value=4294967295, dtype=np.uint32)
```

Comment:
This looks makes sense and looks good.  At some point we might want to note somewhere that the 'idx' in a CommandTable from get_cmds_from_backstop doesn't look much like an index if all the records have the same value?

Reply:
I changed the fill value to be -1, which is more obviously a non-valid value and is likely to provide a clue. It is worth noting that the string repr of a CommandTable does not include the `idx` column, meaning it is considered somewhat of a private attribute.

Reply:
Makes sense.  I should have brought that up as a question yesterday (the "can we spare a bit for the sign and use -1" question or some such).